app/assets/javascripts/sequence_show.js

```diff
@@ -512,3 +535,33 @@ function init_sequence_show(data) {
         }
     }
 }
+
+function toggle_column(col) {
```

Comment:
Ok got it, but does this mean i should move these 2 function within the init_sequence_show?
or relocate them somewhere else?


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3474) by calmalak on Mon Nov 09 2015 at 22:35._